### ISSUE TITLE
Landing Page + Content

### ISSUE THREAD

**Issue by thescientist13** [AnalogStudiosRI/www.blissfestri.com#14]
## Summary
Will want to compose all the parts together into the final page
<img width="779" alt="Screen Shot 2023-03-05 at 3 24 52 PM" src="https://user-images.githubusercontent.com/895923/222983984-00722441-0609-4f52-aa7a-ddff2a176f87.png">

## Details
1. [ ] Combine header, sign up, and event details components on to the page (after #8)
1. [ ] Feedback
    - make logos and location text bigger (add Analog Studios location in italics and in accent color)
    - make header a little bit bigger
    - header doesn't seem centered?

## Nice to have (new issues?)

_Get this text to stretch across breakpoints to like in the designs_
![Screen Shot 2023-04-07 at 9 21 40 AM](https://user-images.githubusercontent.com/895923/230616232-ab38c18e-d562-4713-be05-5b05b23fbc9b.png)
